docs/library.md

```diff
@@ -58,6 +58,7 @@ Arguments include:
   - `graphiqlRoute`: The endpoint the GraphiQL query interface will listen on (**NOTE:** GraphiQL will not be enabled unless the `graphiql` option is set to `true`). Defaults to `/graphiql`.
   - `pgDefaultRole`: The default Postgres role to use. If no role was provided in a provided JWT token, this role will be used.
   - `jwtSecret`: The secret for your JSON web tokens. This will be used to verify tokens in the `Authorization` header, and signing JWT tokens you return in procedures.
+  - `jwtOptions`: The options for the `jwt.verify(jwtToken, jwtSecret, jwtOptions)` call. See the [jsonwebtoken documentation](https://github.com/auth0/node-jsonwebtoken#jwtverifytoken-secretorpublickey-options-callback) for further details.
```

Comment:
Awesome! I just think this needs to be updated and I’ll merge and release ASAP 😊